src/ngraph_encapsulate_op.cc

```diff
@@ -69,9 +70,27 @@ class NGraphEncapsulateOp : public tf::OpKernel {
     // Create the backend
     if (m_ng_backend == nullptr) {
 #if defined(NGRAPH_EMBEDDED_IN_TENSORFLOW)
-      m_ng_backend = std::make_shared<ng::runtime::interpreter::INTBackend>();
+    	m_ng_backend = ng::runtime::Backend::create("INTERPRETER");
+#elif defined(NGRAPH_TF_BACKEND)
+        auto env_value = std::string(std::getenv("NGRAPH_TF_BACKEND")); 
```

Comment:
Avoid `getenv` multiple times, save the "backend_string" value as a member of the class during class construction.

Reply:
I'm not sure if this is still relevant given https://github.com/NervanaSystems/ngraph-tf/blob/8c7c303852fba4382164842083235ded7bbbb17d/src/ngraph_encapsulate_op.cc#L304

Reply:
the fix you had exactly addressed this problem, nice fix